src/libexec/ldapmoduleconfupdate

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+#
+# SPDX-License-Identifier: GPL-2.0-only
+
+set -e
+
+TARGET_MODULE_CONF_FILE="/opt/zextras/data/ldap/config/cn=config/cn=module{0}.ldif"
+SOURCE_MODULE_CONF_FILE="/opt/zextras/common/etc/openldap/zimbra/config/cn=config/cn=module{0}.ldif"
+BACKUP_DIR="/opt/zextras/data/ldap-post-install-backup/"
+BACKUP_MODULE_CONF_FILE="$BACKUP_DIR/cn=module{0}.ldif.bak"
+
+backup_module_config() {
+    echo "* Creating backup of module config file at $BACKUP_MODULE_CONF_FILE"
+    su - zextras -c "mkdir -p $BACKUP_DIR"
+    cp "$TARGET_MODULE_CONF_FILE" "$BACKUP_MODULE_CONF_FILE"
+}
+
+update_module_config() {
+    echo "* Updating module config $TARGET_MODULE_CONF_FILE"
+    cp "$SOURCE_MODULE_CONF_FILE" "$TARGET_MODULE_CONF_FILE"
+}
+
+main() {
+    if [ ! -f "$TARGET_MODULE_CONF_FILE" ]; then
+        echo "Error: Target module config file $TARGET_MODULE_CONF_FILE does not exist."
+        exit 1
+    fi
+
+    if [ ! -f "$SOURCE_MODULE_CONF_FILE" ]; then
+        echo "Error: Source module config file $SOURCE_MODULE_CONF_FILE does not exist."
+        exit 1
+    fi
+
+    if ! diff "$TARGET_MODULE_CONF_FILE" "$SOURCE_MODULE_CONF_FILE" > /dev/null; then
```

Comment:
`diff` requires `diffutils` package installed. is there another way to achieve this, with more common and already used commands?

Reply:
also `cmp` is a diffutils binary

Reply:
And the `diffutils` do not come preinstalled on our supported distributions? Do you have any other suggestions @M0Rf30

Reply:
md5sum, it's part of coreutils

Reply:
@M0Rf30 I added `diffutils` in `carbonio-core` as dependency. We should be fine using `cmp` now. Can you review the changes?